src/store.ts

```diff
@@ -107,6 +107,17 @@ export default class Store extends AinftBase {
     return this.sendRequest(HttpMethod.DELETE, trailingUrl, query);
   }
 
+  giveItemToUser(appId: string, userId: string, itemName: string, quantity: number, reason?: string) {
```

Comment:
minor) Could you add new lines between functions params?

Reply:
Fixed, thanks!